### PR TITLE
193 add notebooks to docs

### LINKED ISSUE #193: Include notebooks of stream and multi-annot scenarios in documentation
- [ ] stream
- [ ] multi-annot

### PR DESCRIPTION
**Pull request by tpham93**
Fixes #193 and #194

### CODE REVIEW

**Review by dakot — CHANGES_REQUESTED** [review]
remove binaries and exp results

File: skactiveml/utils/_visualisation.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+import matplotlib.pyplot as plt
+from matplotlib.colors import Colormap
+from matplotlib.figure import Figure
+from sklearn.utils import check_array
+
+from skactiveml.utils import check_scalar, check_type
+
+
+def check_bound(bound=None, X=None, ndim=2, epsilon=0,
+                bound_must_be_given=False):
```
Comment (mherde) [inline skactiveml/utils/_visualisation.py:13]: @LukasLuehrs `check_bound` should be part of the validation file.

File: skactiveml/utils/_visualisation.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
+import warnings
```
Comment (mherde) [inline skactiveml/utils/_visualisation.py:1]: @LukasLuehrs Move helper functions (except for the check_bound) to the visualization package. Possible filename: `skactiveml.visualization._auxiliary_functions.py`.

File: skactiveml/utils/tests/test_visualisation.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import unittest
```
Comment (mherde) [inline skactiveml/utils/tests/test_visualisation.py:1]: @LukasLuehrs Adjust test according to changes proposed in `skactiveml.utils._visualisation.py`.

**Review by mherde — CHANGES_REQUESTED** [review]

**Review by dakot — APPROVED** [review]

**Review by dakot — CHANGES_REQUESTED** [review]
What about the changes mentioned by mherde?

**Review by dakot — APPROVED** [review]

File: README.rst [hunk 4]
```diff
@@ -59,18 +59,23 @@ The following code implements an active learning cycle with 20 iterations using
...
-         clf.fit(X, y)
-         unlbld_idx = unlabeled_indices(y)
-         X_cand = X[unlbld_idx]
-         query_idx = unlbld_idx[qs.query(X_cand=X_cand, clf=clf)]
-         y[query_idx] = y_true[query_idx]
+            clf.fit(X, y)
+            unlbld_idx = unlabeled_indices(y)
+            X_cand = X[unlbld_idx]
+            query_idx = unlbld_idx[qs.query(X_cand=X_cand, clf=clf)]
+            y[query_idx] = y_true[query_idx]
```
[the remark this answers is not in the document]
Comment (tpham93) [reply README.rst]: removed those spaces and changed the indentation to 4 spaces per indentation

File: README.rst [hunk 5]
```diff
@@ -59,18 +59,23 @@ The following code implements an active learning cycle with 20 iterations using
     X, y_true = make_classification(random_state=0)
     y = np.full(shape=y_true.shape, fill_value=MISSING_LABEL)
 
+    # LogisticRegression needs ititial training data otherwise a warning will
```
[the remark this answers is not in the document]
Comment (tpham93) [reply README.rst]: changed "ititial" to initial

File: skactiveml/visualization/_feature_space.py [hunk 6]
```diff
@@ -4,14 +4,13 @@
...
 from matplotlib.axes import Axes
-from matplotlib.colors import Colormap
 from sklearn.base import ClassifierMixin
 from sklearn.neighbors import KNeighborsRegressor
 
 from ..base import QueryStrategy
 from ..utils import check_scalar
 from ..utils._validation import check_type
-from ..utils._visualisation import mesh, check_bound, _get_boundary_args, \
+from skactiveml.visualization._auxiliary_functions import mesh, check_bound, _get_boundary_args, \
```
Comment (mherde) [inline skactiveml/visualization/_feature_space.py:13]: relative import should be used, i.e., `..visualization._auxiliary_functions import ...`

File: skactiveml/visualization/multi/_multi_annot_feature_space.py [hunk 7]
```diff
@@ -8,7 +8,7 @@
 from skactiveml.utils import is_labeled, check_scalar
 from .._feature_space import plot_decision_boundary
 from ...utils._validation import check_type
-from ...utils._visualisation import mesh, check_bound, _get_contour_args, \
+from skactiveml.visualization._auxiliary_functions import mesh, check_bound, _get_contour_args, \
```
Comment (mherde) [inline skactiveml/visualization/multi/_multi_annot_feature_space.py:11]: relative import should be used, i.e., ..visualization._auxiliary_functions import ...

File: tutorials/multiple_annotators_getting_started.ipynb [hunk 8]
```diff
@@ -5,7 +5,7 @@
    "id": "e1239f66",
    "metadata": {},
    "source": [
-    "# Multi-annotators Pool-based Active Learning With  - Getting Started"
+    "# Pool-based Active Learning With Multi-annotators - Getting Started"
```
Comment (mherde) [inline tutorials/multiple_annotators_getting_started.ipynb:8]: Multi-annotator Pool-based Active Learning - Getting Started

**Review by mherde — CHANGES_REQUESTED** [review]

File: tutorials/multiple_annotators_getting_started.ipynb (hunk 8 above)
Comment (mherde) [reply tutorials/multiple_annotators_getting_started.ipynb:8]: Alternatively: Pool-based Active Learning with Multiple Annotators

**Review by mherde — APPROVED** [review]

**Pull request merged**